SpatialGDK/Source/SpatialGDK/Private/Interop/WellKnownEntitySystem.cpp

```diff
@@ -8,34 +8,37 @@ DEFINE_LOG_CATEGORY(LogWellKnownEntitySystem);
 
 namespace SpatialGDK
 {
-WellKnownEntitySystem::WellKnownEntitySystem(const FSubView& SubView, USpatialReceiver* InReceiver, USpatialWorkerConnection* InConnection,
+WellKnownEntitySystem::WellKnownEntitySystem(const FSubView& InGdkSubView, const FSubView& InSystemEntitySubView,
+											 USpatialReceiver* InReceiver, USpatialWorkerConnection* InConnection,
 											 const int InNumberOfWorkers, SpatialVirtualWorkerTranslator& InVirtualWorkerTranslator,
 											 UGlobalStateManager& InGlobalStateManager)
-	: SubView(&SubView)
+	: GdkSubView(&InGdkSubView)
+	, SystemEntitySubView(&InSystemEntitySubView)
 	, Receiver(InReceiver)
 	, VirtualWorkerTranslator(&InVirtualWorkerTranslator)
 	, GlobalStateManager(&InGlobalStateManager)
 	, Connection(InConnection)
 	, NumberOfWorkers(InNumberOfWorkers)
+	, SnapshotPartitionAuthServerCrashed(false)
```

Comment:
bSnapshotPartitionAuthServerCrashed

It's also not clear from the name that this gets claimed after it's processed (ie. only that the crash in progress)
What about bSnapshotPartitionAuthServerCrashInProgress?